src/main/java/edu/wisc/my/messages/controller/MessagesController.java

```diff
@@ -31,6 +35,33 @@
                 response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
             }
         }
+
+    @RequestMapping(value="/datedMessages", method=RequestMethod.GET)    
+    public @ResponseBody void getDatedMessages(HttpServletRequest request,
```

Comment:
Do we need a separate endpoint for all the message filtering, or should this just be a feature of all messages?

I think a lot of this (business logic) can be moved into the service and leave the controller just for the delivering content purpose.